app/lib/client/methods/sendMessage.js

```diff
@@ -12,6 +12,10 @@ Meteor.methods({
 		if (!Meteor.userId() || s.trim(message.msg) === '') {
 			return false;
 		}
+		const messageAlreadyExists = message._id && ChatMessage.findOne({ _id: message._id });
+		if (messageAlreadyExists) {
+			return;
```

Comment:
maybe on the client it should show a toast message on this case? might be better than doing nothing